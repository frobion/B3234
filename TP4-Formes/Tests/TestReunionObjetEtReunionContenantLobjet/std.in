R Rectangle1 1 3 6 0
R Rectangle2 1 3 8 -4
OR Reunion1 Rectangle1 rectangle2
OR Reunion2 Reunion1 rectangle2
LIST
EXIT

