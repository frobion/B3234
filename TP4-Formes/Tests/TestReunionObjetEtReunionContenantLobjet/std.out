OK
OK
OK
OK
R Rectangle1 1 3 6 0
R Rectangle2 1 3 8 -4
R _Rectangle1 1 3 6 0
R _Rectangle2 1 3 8 -4
OR Reunion1 _Rectangle1 _Rectangle2
DELETE _Rectangle1 _Rectangle2
R _Rectangle1 1 3 6 0
R _Rectangle2 1 3 8 -4
OR _Reunion1 _Rectangle1 _Rectangle2
DELETE _Rectangle1 _Rectangle2
R _Rectangle2 1 3 8 -4
OR Reunion2 _Reunion1 _Rectangle2
DELETE _Reunion1 _Rectangle2


